plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -519,6 +519,33 @@ public void testUnsupportedColumnType()
         session.execute("DROP KEYSPACE keyspace_6");
     }
 
+    @Test
+    public void testNullAndEmptyTimestamp()
+    {
+        String keyspace = "keyspace_7";
+        String tableName = "test_empty_timestamp";
+
+        session.execute("DROP KEYSPACE IF EXISTS " + keyspace);
```

Comment:
Do we need a new keyspace? some other tests just create a new table in existing keyspace.

Reply:
A new keyspace isn't due. Let me change.